scene/resources/visual_shader.cpp

```diff
@@ -442,7 +442,7 @@ void VisualShaderNodeCustom::update_ports() {
 			for (int i = 0; i < output_port_count; i++) {
 				Port port;
 				if (!GDVIRTUAL_CALL(_get_output_port_name, i, port.name)) {
-					port.name = "out" + itos(i);
+					port.name = "Out " + itos(i);
```

Comment:
As an addition, If you touched these port names - you need to change the documentation of `VisualShaderNodeCustom`. This change is kinda meh - I would touch it only if all other input/output port names of other nodes are changed as well (for consistency).

Reply:
I've reverted that change now. This PR only changes VisualShaderNodeOutput again. The other shader nodes can be done in another PR if needed.